### PR TITLE
SD-738: Cookie Preference cookie not listed on cookie policy

### PR DESCRIPTION
**Pull request by SarahJaneLuff**
**Jira Ticket:** https://collaboration.homeoffice.gov.uk/jira/browse/SD-738

**Changes**
	
- Changed cookies.md file back to original, this is now not being used.
- Deleted cookies.html, as we are now using hof-template-partials for the cookie policy.
- Added sessionCookieTable to index.js to insure we are using local cookie names referenced from cookies.json.
- Updated hof-template-partials to version 5.3.4, to add seen_cookie_message into the cookie policy page.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -32,6 +34,7 @@ app.use((req, res, next) => addGenericLocals(req, res, next));
 
 app.use('/cookies', (req, res, next) => {
   res.locals = Object.assign({}, res.locals, req.translate('cookies'));
+  res.locals['session-cookies-table'] = sessionCookiesTable['session-cookies-table'];
```
Comment (sulthan-ahmed) [inline index.js:37]: 👍 this is the thing that @cmedlandho and @MaxOSullivan might have needed for modern-slavery but we missed
Great spot @SarahJaneLuff

File: apps/end-tenancy/views/content/cookies.md [hunk 2]
```diff
@@ -29,10 +33,11 @@ Google Analytics stores information about:
...
-| _ga  | Used to distinguish users     | 2 years    |
-| _gat | Used to throttle request rate | 10 minutes |
+| Name |    Purpose                         |  Expires   |
+|:-----|:-------------------------------    |:---------  |
+| _ga  | Used to distinguish users          | 2 years    |
+| _gat | Used to throttle request rate      | 10 minutes |
+| _gid | Used to count and track page views | 24 hours   |
 
 <div class="panel-indent">
   <p>Google isn’t allowed to use or share our analytics data.</p>
```
Comment (sulthan-ahmed) [inline apps/end-tenancy/views/content/cookies.md:43]: In the PR it says changing the md because you're not using it, shouldn't you just delete it then?

Comment (SarahJaneLuff) [reply apps/end-tenancy/views/content/cookies.md]: deleted cookies.md

**Review by sulthan-ahmed — APPROVED** [review]
lgtm

**Pull request merged**